### PR TITLE
Cutout transform, dropout, quantized distributions etc.

### PR DESCRIPTION
**Pull request by anlthms**
- Make dropout rate configurable
- Make weight decay configurable
- Set training:validation split to 90:10
- Use quantized uniform for ranges inside sweep.yaml
- Pick the most likely class if none crosses the threshold
- Allow inference on CPU with a GPU-trained model
- Use grad-cam to generate class activation maps from the last
  convolution layer
- Move reporting to a separate notebook
- Change EDA notebook to use seaborn

### CODE REVIEW

**Comment by anlthms** [comment]
report.sh requires `pip install grad-cam` on the compute server (until we update the boot images)

**Comment by hornof** [comment]
- Created code from Template (Kaggle = true), followed instructions from README file
- Attached to a compute server and downloaded data
- Ran eda.ipynb (took 3m22s).  Looks great.
- Ran experiment
  - rsync: read errors mapping "/home/ubuntu/pr14/output/model.pth": No data available (61) (ide-service issue #579)
- Ran report.sh.  Looks great.
  - Had to do "pip install grad-cam", as mentioned
  - Didn't check "Track Experiment" the first time (don't think results.html can be viewed)
  - Worth mentioning report.sh somewhere (maybe in the README file, and suggest checking "Track Experiment")?  
- Ran kaggle.ipnb on Kaggle (took 10m)
  - Included all 3 inputs files, turned off Internet, didn't select GPU
  - Submitted results: scored 0.63747 (431/625)

image

image

**Comment by anlthms** [comment]
ah... good point about mentioning report.sh. I will add it to the README before merging this.
